pages/terra/index.mdx

```diff
@@ -5,4 +5,10 @@ description: Welcome to the documentation for the Terra repository.
 
 # Welcome
 
-With Terra, you can install the latest packages knowing that quality and security are assured. Terra is also a great way to distribute your software on Fedora and its derivatives.
+Terra is a rolling repository RPM repository for Fedora Linux and its derivatives.
+With Terra, you can install the latest packages knowing that quality and security are assured.
+Terra is also a great way to distribute your software on Fedora and its derivatives.
+
+- Website: https://terra.fyralabs.com/
```

Comment:
use md links instead?

Reply:
The point is I want the links to be visible

Reply:
alrightie